src/client/view/searchBar.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export const SlidingSearchBar = ({ parentComp }) => (
+  <div>
+    <div className="ui fluid right icon input">
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2177445439/issues/source?bid=3148782&fileBranchId=3163407#l5)